### PR TITLE
Add Guide Page - Software request process (WIP)

### PR DESCRIPTION
**Pull request by saracope**
This adds a new Guide page for the GSA Software Request Process (as-is).

This is a current work in progress.

### CODE REVIEW

**Comment by saracope** [comment]
Tagging @jfredrickson5, @jcastle, and @afeld for feedback.

File: _guides/software-request-guide.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  <a href="http://codepen.io/saracope/full/aWPePG/" target="_blank"><img src="{{ site.baseurl }}/assets/img/guides/software-request-process-map.png"
+    alt="Software Request Process Map"
+    style="float: left;  margin-right: 10px;"></a>
+
+### Introduction
+
+Occasionally you may need to use software that is not already available to you. We've created this resource to guide you in the process of requesting software at GSA. Follow these simple steps below and view our interactive [process map](http://codepen.io/saracope/full/aWPePG/){:target="_blank"} to get started.
+
+### Research
+Once you have identified a need, discuss this with your supervisor and colleagues to get more information about what tool may be best to use. Be sure check if the software you want is available in the [GSA EA Analytics & Reporting](https://ea.gsa.gov/#!/itstandards) (GEAR) IT Standards list. Note: The GEAR site is only available within the GSA network. Once you have identified the software you would like, you will need to take some time to get all the information you will need to request it.
```
Comment (afeld) [inline _guides/software-request-guide.md]: Some notes here:

* I'd split this last sentence out to a new paragraph.
* We can offer that people can check [the public export of the data](https://github.com/GSA/data/blob/gh-pages/enterprise-architecture/it-standards.csv), if they need to access off the network.
* We may want to explain what the different `Status`es in GEAR mean in terms of how they should be making requests.

File: _guides/software-request-guide.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+<br>
+<br>
+Resources: [GEAR](https://ea.gsa.gov/#!/itstandards)
+
+### Gather
+You will need several pieces of information when completing a software request. Things like purpose, cost, and download link among others. Follow [this checklist](https://docs.google.com/a/gsa.gov/spreadsheets/d/1zIyZGdmEuWUeuko7yEaupETz2YGJQpOumHwiUntul64/edit?usp=sharing){:target="_blank"} to be sure you've gathered all the info you will need.
```
Comment (afeld) [inline _guides/software-request-guide.md]: I'm a big fan of checklists, though the information there is a bit redundant with some on this page. Might consider how to consolidate/deduplicate them.

File: _guides/software-request-guide.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+title: Requesting Software at GSA
+category: Team
+---
+
+  <a href="http://codepen.io/saracope/full/aWPePG/" target="_blank"><img src="{{ site.baseurl }}/assets/img/guides/software-request-process-map.png"
+    alt="Software Request Process Map"
+    style="float: left;  margin-right: 10px;"></a>
```
Comment (afeld) [inline _guides/software-request-guide.md]: Minor: I believe the Security and 508 reviews happen in parallel.

File: _guides/software-request-guide.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+### Request
+Once you have gathered the info you will need on the software, it's time to put in your request. A quick way to do this is to select the Submitting a Service Catalog, [Request for Software](https://gsa.service-now.com/GSA_Self-Service/software.do) option in ServiceNow.
+<br>
+<br>
+Resources: [ServiceNow](https://gsa.service-now.com/GSA_Self-Service/software.do)
+
+#### Non-Standard Software Requests
+If the software you are requesting is not listed as Approved on the IT Standards List in GEAR, it will need to go through the non-standard request approval process. This involves 2 additional reviews where the software is evaluated for Security and Accessibility requirements. After these reviews are complete, the Technology Standards Committee (TSC) will provide their recommendation as to whether or not the software should be added to the IT Standards list. Finally, the Chief Technology Officer (CTO) will approve or reject the request. If approved, the software will be added to the IT Standards List in Gear. This process can take quite a bit longer than requesting software that is already approved in GEAR. Expect 4-6 weeks for software to be completely evaluated.
```
Comment (afeld) [inline _guides/software-request-guide.md]: Minor: `Gear` -> `GEAR` in the third to last sentence.

File: _guides/software-request-guide.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+<ul class="usa-accordion">
+  <li>
+    <button class="usa-accordion-button"
+      aria-expanded="true" aria-controls="Question-1">
+      What is the difference between Standard and Non-standard software?
+    </button>
+    <div id="Question-1" class="usa-accordion-content">
+      <p>
+      Standard software has previously been evaluated and approved to be included in the IT Standards List in GEAR. You must still put in a request for Standard software. This type of request typically takes 1-2 weeks to complete.
+      Non-standard software has not been evaluated for inclusion in the IT Standards List and must complete a full review for Security, 508 Accessibility, and Enterprise Architecture requirements. This review and evaluation process can take 4-6 weeks to complete.
```
Comment (afeld) [inline _guides/software-request-guide.md]: I'd split the estimates out to their own FAQ question, or move them up and make them more prominent within the description/diagram above.

File: _guides/software-request-guide.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    </div>
+  </li>
+  <li>
+    <button class="usa-accordion-button"
+      aria-controls="Question-2">
+      Do I still need to enter a ServiceNow ticket if the software I want is already approved in the IT Standards List?
+    </button>
+    <div id="Question-2" class="usa-accordion-content">
+      <p>
+      Yes, you will still need to enter a ServiceNow ticket to request approved software.
```
Comment (afeld) [inline _guides/software-request-guide.md]: Is it the same type of request?

**Comment by afeld** [comment]
Also, hoping you saw [the doc](https://docs.google.com/document/d/1AUyRggdKWLRcHC3E4ssHE_2tmOaDYTv0ex0lgwbwFSA/edit) I was working on to explain the process to TTS?

File: _guides/software-request-guide.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+#### Non-Standard Software Requests
+If the software you are requesting is not listed as Approved on the IT Standards List in GEAR, it will need to go through the non-standard request approval process. This involves 2 additional reviews where the software is evaluated for Security and Accessibility requirements. After these reviews are complete, the Technology Standards Committee (TSC) will provide their recommendation as to whether or not the software should be added to the IT Standards list. Finally, the Chief Technology Officer (CTO) will approve or reject the request. If approved, the software will be added to the IT Standards List in Gear. This process can take quite a bit longer than requesting software that is already approved in GEAR. Expect 4-6 weeks for software to be completely evaluated.
+
+#### Pilot Software Requests
+If the feasibility or applicability of the software is not known or not yet proven, you may request to evaluate the software on a trial basis. This type of request is know as a Pilot. Pilot software must never be on a production network or contain sensitive federal data and the cost for a Pilot may not exceed $25k.
```
Comment (afeld) [inline _guides/software-request-guide.md]: Suggestions:

* Say something like "the advantage is, the review process is simpler/faster"
* Make these criteria a list
* Make this section [plainer-language](https://content-guide.18f.gov/plain-language/)
    * For example, "If the feasibility or applicability of the software is not known" could be changed to "If you want to try a piece of software but aren't sure if you want it to go through a full review yet, ..."

**Pull request merged**